src/StreamMaster.h

```diff
@@ -137,7 +137,7 @@ template <typename Streamer> class StreamMaster {
   /// stream is in any error state
   const StreamMasterError status() {
     for (auto &s : Streamers) {
-      if (s.second.runStatus().connectionOK()) {
+      if (s.second.runStatus() >= StreamerError::IS_CONNECTED) {
```

Comment:
This is one of the things I forgot to fix yesterday. I changed the name of the original type from `StreamerError` into `StreamerStatus` but forgot to change a `using` statement. I have since fixed that.

Reply:
Oh, looks like I have to fix that for `StreamMaster` as well.